qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -259,3 +256,9 @@ def _incremental_search(self, text):
 
         if self.prefix() in ['/', '?']:
             self.got_cmd[str].emit(search_prefixes[text[0]] + text[1:])
+
+    @pyqtSlot(str)
+    def _exit_prefix(self, text):
+        if not text or text[0] not in modeparsers.STARTCHARS:
```

Comment:
I don't think we should exit command mode when deleting the prefix but there's still text - vim, for example, doesn't either (it just disallows deleting it).